src/DynamoCore/Library/TypedParameter.cs

```diff
@@ -20,6 +21,8 @@ public TypedParameter(string parameter, ProtoCore.Type type, AssociativeNode def
             Name = parameter;
             Type = type;
             DefaultValue = defaultValue;
+
+            defaultValueString = shortArgumentName;
```

Comment:
It was using `defaultValue.ToString()` earlier but `defaultValue` in this case is the AST with the fully qualified class name. I need to use the short name of the original AST that is passed into the ctor.